Source/Bogus/Faker[T].cs

```diff
@@ -246,10 +247,10 @@ private void PopulateInternal(T instance, string[] ruleSets)
                     }
                 }
 
-                foreach( var ruleSet in ruleSets )
+                foreach (var ruleSet in ruleSets)
```

Comment:
Hey @gdimauro , for future reference, all these space formatting edits add a lot of noise to the PR.